src/config/graphical-session/kde.md

```diff
@@ -17,3 +17,23 @@ instead.
 If you wish to start an X-based session from the console, use
 [startx](./xorg.md#startx) to run `startplasma-x11`. For a Wayland-based
 session, run `startplasma-wayland` directly.
+
+## Dolphin
+
+Dolphin is the default file manager of the KDE desktop environment. It can be
+installed on its own by installing the `dolphin` package, or it can be installed
+as part of the `kde5` or `kde5-baseapps` packages.
```

Comment:
"It can be installed on its own by installing the `dolphin` package or as a dependency of the `kde5-baseapps` meta-package."

`dolphin` isn't a dependency of `kde5`.

Reply:
I've fixed it.